storm-shim/src/main/java/storm/mesos/shims/ICommandLineShim.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package storm.mesos.shims;
+
+/**
+ * Returns the command line, shimmed and 0.9.X and 0.10.X have slightly a different command
+ */
+public interface ICommandLineShim {
+
+  public String getCommandLine();
```

Comment:
I agree it could be improved but I'm not sure it's worth the effort here.  It's a shim, it lives in a very small area of the code and isn't going to be touched again until storm 0.11.0 (if it needs to be).